homeassistant/components/sensor/airvisual.py

```diff
@@ -0,0 +1,304 @@
+"""Define sensors for AirVisual air quality information."""
+
+import asyncio
+from logging import getLogger
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (ATTR_ATTRIBUTION, CONF_API_KEY, CONF_LATITUDE,
+                                 CONF_LONGITUDE, CONF_MONITORED_CONDITIONS)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = getLogger(__name__)
+REQUIREMENTS = ['pyairvisual==0.1.0']
+
+ATTR_CITY = 'city'
+ATTR_COUNTRY = 'country'
+ATTR_POLLUTANT_SYMBOL = 'pollutant_symbol'
+ATTR_POLLUTANT_UNIT = 'pollutant_unit'
+ATTR_STATE = 'state'
```

Comment:
Import from const.py.

Reply:
I was under the impression that `const.py` was for generally-useful constants, and that attributes applying to certain platforms only should exist in those platform files. I could see country and state being generally-useful, but what about the two pollutant ones?

Reply:
You're correct, you shouldn't add non general constants to const.py. I'm only talking about `ATTR_STATE` which already exists in const.py.

Reply:
Ah, got it! Will do!